package.json

```diff
@@ -30,7 +30,7 @@
     "vuex": "^3.0.1"
   },
   "devDependencies": {
```

Comment:
I needed to bump the Vue test utils to use the `setSelected()` helper. This is currently causing a bunch of warnings related to the rename of `shallow()` to `shallowMount()`. I'll have a follow-up PR to fix those.